sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/integrationpoint/IntegrationPointExampleNode.kt

```diff
@@ -129,10 +125,10 @@ class IntegrationPointExampleNode(buildContext: BuildContext) : Node(buildContex
             }
     }
 
-    private fun observeActivityResult() {
-        activityResultsCancellable = activityStarter
-            .events(this)
-            .observe { event ->
+    private suspend fun observeActivityResult() {
+        activityStarter
+            .events(this@IntegrationPointExampleNode)
+            .collect { event ->
```

Comment:
can you please fix `requestCodeClientId`. It will crash after requesting permissions + rotate screen.

`override val requestCodeClientId: String = IntegrationPointExampleNode::class.java.name`